### DISCUSSION TITLE
Mechanism of relay

### DISCUSSION THREAD

**Discussion by foxB612** [mastodon/mastodon#18792] [Q&A]
Hello, can anyone explain how relay works in mastodon? Details about the communication protocol will be much appreciated, for example, to which path a instance should post to the relay, what content it should send, and whether/when the relay (and other instances) should verify the content. (Assume that authorized_fetch is not in consideration)

I thought there should be detailed documentations on this, which I failed to find. Would be glad if anyone just throws a link to me.

**Comment by kzxpr** [comment]
Any news to this? I've been wondering about this as well.

EDIT: Maybe it's an ActivityPub standard, actually
